Sources/ContainerClient/Core/Volume.swift

```diff
@@ -46,13 +50,26 @@ public struct Volume: Sendable, Codable, Equatable, Identifiable {
         labels: [String: String] = [:],
         options: [String: String] = [:],
         sizeInBytes: UInt64? = nil,
+        isAnonymous: Bool = false,
```

Comment:
I think I'd prefer that we don't do what Docker does, at least for now:

> Just like named volumes, anonymous volumes persist even if you remove the container that uses them, except if you use the --rm flag when creating the container, in which case the anonymous volume associated with the container is destroyed.

IMO it introduces a bunch of server-side complexity that we can add later should we need it. Perhaps we can approach the problem of pruning such volumes creatively to make up for the lack of the `--rm` anonymous volume feature.